includes/class-wc-abstract-google-analytics-js.php

```diff
@@ -107,6 +107,20 @@ public function add_transaction( $order ) {
 	 */
 	abstract protected function add_transaction_enhanced( $order );
 
+	/**
+	 * Get item identifier from product data
+	 *
+	 * @param  WC_Product $product WC_Product Object
+	 * @return stirng
+	 */
+	public static function get_product_identifier( $product ) {
+		if ( ! empty( $product->get_sku() ) ) {
+			return esc_js( $product->get_sku() );
+		} else {
+			return esc_js( '#' . ( $product->is_type( 'variation' ) ? $product->get_parent_id() : $product->get_id() ) );
```

Comment:
Currently, the variation ID is sent on the checkout started event but it isn't used for the add to cart event. The data structure will have to change for some events so that we have consistency across all tracking.

Given that we're also specifying the `variant` when sending I don't think it makes sense to use the variation ID as the item ID which is why I left it out here.